cw-relayer/tests/e2e/docker/relayer.Dockerfile

```diff
@@ -0,0 +1,74 @@
+# syntax=docker/dockerfile:1
+# https://github.com/osmosis-labs/osmosis/blob/v12.3.0/Makefile
+# Modified to use the apline image instead of distroless and include bash
```

Comment:
Are these comments relevant? if so let's make them more clear
```suggestion

# Based on the osmosis makefile, updated to use the alpine image 
# instead of distroless and include bash.
# Ref: https://github.com/osmosis-labs/osmosis/blob/v12.3.0/Makefile
```